src/assignment/assignment.service.ts

```diff
@@ -340,6 +340,8 @@ export class AssignmentService {
 		delete (assignmentToSave as any).__typename;
 		delete (assignmentToSave as any).descriptionRichEditorState;
 		delete assignmentToSave.contributors;
+		delete assignmentToSave.quality_labels;
```

Comment:
should this still be deleted now that the insert/update logic has moved to the backend?
I think the backend route expects the quality labels to be passed and handles them inside the route.